src/target/source/codegen_c.cc

```diff
@@ -648,6 +648,21 @@ void CodeGenC::VisitExpr_(const CallNode* op, std::ostream& os) {  // NOLINT(*)
       os << " != ";
       this->PrintExpr(op->args[0], os);
       os << ")";
+    } else if (op->op.as<OpNode>()->name == "tir.round") {
```

Comment:
[Suggestion] consider using op->op.same_as(Op::Get("tir.round")) and similar to the following

Reply:
This is a great point. Additionally, we can cache the operators in the class member, so the original lookup only happens once in initialization time.

See examples in
https://github.com/apache/tvm/blob/main/src/target/llvm/codegen_llvm.h#L352

Reply:
Please also see my other points (move to handling intrinsic lowering in C backend instead of the codegen.

Reply:
> This is a great point. Additionally, we can cache the operators in the class member, so the original lookup only happens once in initialization time.
> 
> See examples in
> https://github.com/apache/tvm/blob/main/src/target/llvm/codegen_llvm.h#L352

@tqchen 

Like [```op->op.same_as(builtin::isnan())```](https://github.com/apache/tvm/blob/2365c7ee6620f672c172247afc73da3884165884/src/target/source/codegen_c.cc#L645) , right ?